addons/website_sale/controllers/main.py

```diff
@@ -495,22 +497,40 @@ def old_product(self, product, category='', search='', **kwargs):
         # Compatibility pre-v14
         return request.redirect(_build_url_w_params("/shop/%s" % request.env['ir.http']._slug(product), request.params), code=301)
 
-    @route(['/shop/product/extra-images'], type='jsonrpc', auth='user', website=True)
-    def add_product_images(self, images, product_product_id, product_template_id, combination_ids=None):
+    @route(['/shop/product/extra-media'], type='jsonrpc', auth='user', website=True)
+    def add_product_media(self, media, type, product_product_id, product_template_id, combination_ids=None):
         """
-        Turns a list of image ids refering to ir.attachments to product.images,
+        Handles adding both images and videos to product variants or templates,
         links all of them to product.
         :raises NotFound : If the user is not allowed to access Attachment model
```

Comment:
Please take time to write a better docstring as you are changing it 
```suggestion
        :param type: [...] can be either image or video
        :raises NotFound : If the user is not allowed to access Attachment model
```